restapi.go

```diff
@@ -790,6 +790,8 @@ func (s *Session) GuildMember(guildID, userID string) (st *Member, err error) {
 	}
 
 	err = unmarshal(body, &st)
+	// The returned object doesn't have the GuildID attribute so we will set it here.
+	st.GuildID = guildID
```

Comment:
Good catch! Not sure tho if it would be the right PR to include the fix.

Reply:
This is actually needed for the `*Member.AvatarURL` method, else it wouldn't be able to form the URL https://github.com/bwmarrin/discordgo/pull/1077/files#diff-ed32211335d38b2d817f745b55cd71fcc6f824cc578a680cdbed3912002876b6R829, but I can move it if necessary

Reply:
Oh, then it's all good. No need to move it to different PR.